src/widgets/JSONForm/widgets/Checkboxes.js

```diff
@@ -19,16 +19,23 @@ const selectValue = (value, selected, all) => {
 
 const deselectValue = (value, selected) => selected.filter(v => v !== value);
 
-export const Checkboxes = ({ disabled, onChange, options, readonly, value: checkboxesValue }) => {
+export const Checkboxes = ({
+  disabled,
+  onChange,
+  options,
+  readonly,
+  value: checkboxesValue,
+  onBlur,
+  id,
+}) => {
   const { enumOptions, enumDisabled } = options;
 
   const _onChange = option => checked => {
-    if (checked) {
-      const all = enumOptions.map(({ value }) => value);
-      onChange(selectValue(option.value, checkboxesValue, all));
-    } else {
-      onChange(deselectValue(option.value, checkboxesValue));
-    }
+    const all = enumOptions.map(({ value }) => value);
+    const newValue = checked
+      ? onChange(selectValue(option.value, checkboxesValue, all))
+      : onChange(deselectValue(option.value, checkboxesValue));
+    onBlur(id, newValue);
```

Comment:
in all of these widgets, not necessary to add the value to the onBlur - the method on the form is expecting it though, so I thought best to be complete just in case this is used later.